chain/bitcoind_client.go

```diff
@@ -906,8 +914,6 @@ func (c *BitcoindClient) FilterBlocks(
 // the client in the watch list. This is called only within a queue processing
 // loop.
 func (c *BitcoindClient) rescan(start chainhash.Hash) error {
-	log.Infof("Starting rescan from block %s", start)
```

Comment:
is this log not useful?

Reply:
It's already logged by the wallet itself, so opted to remove this to avoid the duplicate info.